tests/input/cs/exception-filters.cs

```diff
@@ -24,7 +24,7 @@ void TestExceptionFilter()
 }
 try {
   int a = (int)when.foo();
-} catch (Exception e) when (DateTime.Now.DayOfWeek == DayOfWeek.Saturday)
+} catch (Exception e)when (DateTime.Now.DayOfWeek == DayOfWeek.Saturday)
```

Comment:
The above `CT_SPAREN_CLOSE` change seems correct. 
I don't understand why the space was removed here. Did you forget to push a spacing option for the `ben_036.cfg` file and a corresponding test output?

Reply:
The line 1162 at _space.cpp_ was not covered.
The rule is "FORCE".
I changed the test data _exception-filters.cs_ to prove it is now working.